test/MOI_wrapper_bridged.jl

```diff
@@ -4,60 +4,23 @@ const MOIB = MOI.Bridges
 const MOIT = MOI.Test
 
 const BRIDGED = MOIB.full_bridge_optimizer(SCIP.Optimizer(display_verblevel=0), Float64)
-const CONFIG = MOIT.TestConfig(atol=1e-5, rtol=1e-5, duals=false, infeas_certificates=false)
-const CONFIG3 = MOIT.TestConfig(atol=1e-3, rtol=1e-2, duals=false, infeas_certificates=false)
-
-@testset "MOI Continuous Linear" begin
-    excluded = [
-        "linear1",  # needs MOI.delete (of variables in constraints)
-        "linear5",  # needs MOI.delete (of variables in constraints)
-        "linear11", # broken in SCIP (#2556)
-        "linear13", # TODO: support MOI.FEASIBILITY_SENSE
-        "linear14", # needs MOI.delete (of variables in constraints)
-    ]
-    MOIT.contlineartest(BRIDGED, CONFIG, excluded)
-end
-
-@testset "MOI Continuous Conic" begin
-    MOIT.lintest(BRIDGED, CONFIG)
-
-    # SOC tests fail because of lower bound requirement of RHS var.
-    # MOIT.soctest(BRIDGED, CONFIG)
-    # MOIT.rsoctest(BRIDGED, CONFIG)
-
-    # other cones not supported
-    # MOIT.geomeantest(BRIDGED, CONFIG)
-    # MOIT.exptest(BRIDGED, CONFIG)
-    # MOIT.sdptest(BRIDGED, CONFIG)
-    # MOIT.logdettest(BRIDGED, CONFIG)
-    # MOIT.rootdettest(BRIDGED, CONFIG)
-end
-
-@testset "MOI Quadratic Constraint" begin
-    # needs objective bridge (MOI/#529)
-    # MOIT.qptest(BRIDGED, CONFIG)
-
-    MOIT.qcptest(BRIDGED, CONFIG)
-    MOIT.socptest(BRIDGED, CONFIG3)
-end
-
-@testset "MOI Integer Linear" begin
-    excluded = String["semiconttest", "semiinttest"]
-    MOIT.intlineartest(BRIDGED, CONFIG, excluded)
-    MOIT.indicator3_test(BRIDGED, CONFIG)
-end
-
-@testset "MOI Integer Conic" begin
-    # SOC tests fail because of lower bound requirement of RHS var.
-    # MOIT.intconictest(BRIDGED, CONFIG)
-end
-
-@testset "MOI NLP" begin
-    # None of tests provide expression graphs in the evaluator.
-    # MOIT.nlptest(BRIDGED, CONFIG)
-end
-
-@testset "MOI Unit tests" begin
-    # TODO: most tests need get-variable-by-name etc.
-    # MOIT.unittest(BRIDGED, CONFIG)
+const CONFIG_BRIDGED = MOIT.Config(atol=1e-5, rtol=1e-5, exclude=Any[
+    MOI.ConstraintDual, MOI.ConstraintName, MOI.VariableName, MOI.DualObjectiveValue, MOI.VariableBasisStatus, MOI.ConstraintBasisStatus,
+])
+
+@testset "MOI unit tests" begin
+    excluded = copy(MOI_BASE_EXCLUDED)
+    append!(excluded, [
+        "test_linear_Interval_inactive",
+        "test_linear_integration",
+        "test_model_ordered_indices", # TODO should fix? ListOf in order of creation
```

Comment:
Yes, this needs to be fixed.

I usually add a comment why we're excluding tests as well.  `test_linear_integration` is a fairly important test, so it'd be nice to know why it's skipped.

Reply:
fixed here. I added an explanation for the integration test, deleting a variable in SCIP is only allowed in certain stages so we are fairly conservative in the wrapper